ambassador/ambassador/envoy/v2/v2route.py

```diff
@@ -44,7 +62,7 @@ def __init__(self, config: 'V2Config', group: IRHTTPMappingGroup) -> None:
         # `per_filter_config` is used for customization of an Envoy filter
         per_filter_config = {}
 
-        if group.get('bypass_auth', False):
+        if mapping.get('bypass_auth', False):
```

Comment:
Hmmm. This seems like a behavior change?

Reply:
Yeah, I've done this to fields that are not touched in the group. Other fields are `host_rewrite`, `auto_host_rewrite`, etc.

Till now `V2Route` was not aware of a mapping (mostly because of the weight), but now that weights are tied per mapping and not per group, I'm getting fields straight from the mapping.